guidelines/mwm-workflow-spec.md

```diff
@@ -283,7 +283,28 @@ For more information about Kubernetes requests & limits, see https://kubernetes.
 
 
 ### Evaluators
-Evaluators are logical statement strings that may be used to determine which tasks are executed. They can make use of the execution context _metadata_.
+Conditional evaluators are logical statement strings that may be used to determine which tasks are executed. They can make use of the execution context _metadata_ and dicom tags. All conditions must evaluate to true in order for the task to be triggered.
+
+#### Supported Evaulators
+Conditional evaluators should support evaluating workflow variables against predefined values with the following operators:
+
```

Comment:
more information on conditionals here:
https://github.com/Project-MONAI/monai-deploy-workflow-manager/blob/develop/guidelines/mwm-conditionals.md

Reply:
Ahh cheers ill provide a link to this